File: algorithms/Codility-master/Codility-master/6_Sorting/Triangle.js
function solution(A) {
    // write your code in JavaScript (Node.js 8.9.4)
    
    if (A.length <= 2) return 0;
    
    let sorA = A.sort(function(a, b){return a - b})
    
    for (let i=0; i<sorA.length - 2; i++ ){
        if (sorA[i]+sorA[i + 1]> sorA[i + 2]) return 1;
    }
                
    return 0
}

File: algorithms/Codility-master/Codility-master/6_Sorting/Triangle.py
def solution(A):
    # write your code in Python 3.6
    if len(A) <= 2 : return 0
    
    sorA = sorted(A)
    
    for i in range(len(sorA) - 2):
        p = sorA[i]
        q = sorA[i + 1]
        r = sorA[i + 2]
        
        if (p+q>r) :
            return 1
                
    return 0
        